dataprofiler/tests/profilers/profiler_options/test_boolean_option.py

```diff
@@ -59,8 +59,6 @@ def test_validate_helper(self):
         # Option is_enabled is not a boolean
         option = self.get_options(is_enabled="Hello World")
         expected_error = ["{}.is_enabled must be a Boolean.".format(optpth)]
-        expected_error += ["{}.{}.is_enabled must be a Boolean." 
-                           .format(optpth, key) for key in self.keys]
```

Comment:
Why did you delete this?

Reply:
bc is_enabled no longer propagates setting everything at every level unless it is specified as "*.is_enabled"

Reply:
Ah I see